docs/manpages/zone2ldap.1.md

```diff
@@ -20,6 +22,15 @@ inserted to an LDAP server.
 --basedn=*DN*
 :    Base DN to store objects below
 
+--create-ent
```

Comment:
ENTs are mandatory in DNS, not just in DNSSEC. We should attempt to do the right thing in all modes that we can.